pykokkos/interface/views.py

```diff
@@ -15,6 +15,8 @@
 from pykokkos.bindings import kokkos
 import pykokkos.kokkos_manager as km
 
+import logging
```

Comment:
@NaderAlAwar can you confirm that you are fine that logging package is used for reporting here

Reply:
We wouldn't need this after removing the call to the logger later.

Reply:
Ok, but that was not my questions: do we want to keep any logging in these files using the logging package.

Reply:
Not sure, I would have to go over how they are currently used to see what changes are necessary